_pages/icml_2020/icml-2020-travel-es.html

```diff
@@ -0,0 +1,12 @@
+---
+layout: main
+permalink: /icml-2020-travel-es
+title: LXAI @ ICML
+published: true
```

Comment:
Until information is updated on the Spanish version of this site, published should be false.